controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/CommitRequestHandler.java

```diff
@@ -88,24 +89,29 @@ public CommitRequestHandler(final StreamMetadataStore streamMetadataStore,
         String scope = event.getScope();
         String stream = event.getStream();
         int epoch = event.getEpoch();
+        long requestId = event.getRequestId();
```

Comment:
having requestId in commit events is not meaningful.

A commit event is posted for each transaction whose commit is requested. 

However, commit event only contains `epoch`. 

When the event is picked for processing, all transactions that have been marked for commit are picked and committed as part of a single commit task. 

So the request id being used here is not really correct reflection of what is intended to be achieved.

Reply:
According to our last discussion, the tracing mechanism has been at the moment removed from transaction operations, as a proper logging of `txnId`in all the logs may provide similar advantages.

Reply:
Please, see #3003.